mmv1/products/logging/LogView.yaml

```diff
@@ -27,17 +27,31 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 # Skipping sweeper since this is a child resource.
 skip_sweeper: true
 description: 'Describes a view over log entries in a bucket.'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  skip_import_test: true
+  parent_resource_type: 'google_logging_log_view'
```

Comment:
Do you know if you need `parent_resource_type`? I think what you've specified here is the default.

Reply:
Done.